### ISSUE TITLE
Command line options aren't available when using init.d script to start Beats

### ISSUE THREAD

**Issue by dedemorton** [elastic/beats#812]
Based on the discussion [here](https://discuss.elastic.co/t/no-debugging-data-returning-after-restarting-beat-with-debugging-logging-turned-on/39807/2), we need to clarify the Beats documentation:

* Make it clear that the command line options aren't supported if you are using the init.d script. Maybe it would be enough to show the full command line: `sudo /usr/bin/topbeat -c ....`
* Possibly update the getting started documentation to show that there is another way to start Topbeat so that you can pass in command line options (need to balance this with desire to avoid making the getting started too long)
* Not related, but I think the section should be called "Starting Topbeat" (not "Running Topbeat") and there should also be a section called "Stopping Topbeat".

**Comment by urso** [comment]
1. TBH, I wouldn't expect init scripts to pass additional CLI parameters to the service. But then Sys-V init system can be quite inconsistent. 

Plus, most systems switching to systemd.

2. starting the beat in foreground in getting started sounds good. Maybe show how to start in debug mode `-d "*"` for checking configuration works.

**Comment by dedemorton** [comment]
@urso That's a good point. Maybe the real solution here is beefing up the documentation around debugging.

**Comment by Adron** [comment]
:+1: to beefing up the debugging documentation. I made the mistake of attempting to pass parameters to the init script - and realized only after getting an answer that I should pass them to the executable. I've run into that in a few places myself where I stumble through debugging. Leading to these two postings I made:

* https://discuss.elastic.co/t/understanding-testing-installation-setup-of-topbeats-beats/39176
* https://discuss.elastic.co/t/no-debugging-data-returning-after-restarting-beat-with-debugging-logging-turned-on/39807

I'm game for contributing to the docs too, but am curious about some guidance where a "debugging" section may go. I've submitted some PRs already regarding some of the Getting Started sections for the various Beats too.

**Comment by dedemorton** [comment]
@Adron Thanks for your interest in contributing to the doc. If you have content that you'd like to see in the doc, it would be best at this point to open an issue against the doc and include the details that you'd like to see added. I'm in the process of making some structural changes and expanding the Beats reference guides to include more guidance info (around things like debugging, performance tuning, and so on). I wouldn't want you to spend time duplicating effort on something I already have underway. But of course, any form of contribution is welcome.

**Comment by Adron** [comment]
That's excellent @dedemorton. I'll try to grab a little time this week and put a list of things that would be helpful together and open an issue as you've mentioned. I'm guessing it'll be a day or two, but I'll at least try to put together a list of things I'd like to see in the docs.  :+1:

**Comment by dedemorton** [comment]
Closing in favor of tracking this issue in https://github.com/elastic/beats/issues/2482
